### PR TITLE
Added sherlock explanations and questions

### PR DESCRIPTION
**Pull request by mviswanathsai**

### CODE REVIEW

File: src/questions/sherlock.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
+- uuid: e00d2b19-5495-4ce5-87c2-18b9df7f6658
+  type: SingleChoice
+  content: What is Sherlock?
+  hint: No Hint
```
Comment (RobinNagpal) [inline src/questions/sherlock.yaml]:
```suggestion
  hint: NoHint
```

Change at other places also

File: src/questions/sherlock.yaml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      key: B
+    - content: Mutual funds company
+      key: C
+    - content: Bank
+      key: D
+- uuid: cdc68343-ec3e-4fee-a1a8-2807dd0bd714
+  type: SingleChoice
+  content: To whom does Sherlock provide insurance cover?
+  hint: No Hint
+  explanation: Sherlock provides insurance cover only to the smart contracts that are audited by Sherlock.
```
Comment (RobinNagpal) [inline src/questions/sherlock.yaml]:
```suggestion
  explanation: Sherlock provides insurance cover only to protocols and covers loss of fund by smart contracts that are audited by Sherlock.
```

File: src/questions/sherlock.yaml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  explanation: Sherlock will charge the protocol for a 4 million USD cover and not a 5 million USD cover.
+  answerKeys:
+    - B
+  difficultyLevel: "Medium"
+  choices:
+    - content: Sherlock will charge the protocol for a 5 million USD cover
+      key: A
+    - content: Sherlock will charge the protocol by for a 4 million USD cover and not a 5 million USD cover
+      key: B
+    - content: Sherlock will charge the protocol for ``5 million + Sherlock's TVL / 2``
```
Comment (RobinNagpal) [inline src/questions/sherlock.yaml]: Did you cross check the formatting of this?

File: src/questions/sherlock.yaml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+      key: B
+    - content: Sherlock will charge the protocol for ``5 million + Sherlock's TVL / 2`` 
+      key: C
+    - content: None of these
+      key: D
+- uuid: a0f302e4-1097-4a69-afdc-799b09e5f530
+  type: SingleChoice
+  content: Up to how much of the user's staked USDC is slashed in the event of an insurance claim? 
+  hint: No Hint
+  explanation: On staking, the user accepts the risk that up to 50% of their staked USDC could be slashed in the case of an attack (claim).
```
Comment (RobinNagpal) [inline src/questions/sherlock.yaml]: Upto 50% for a single claim?

File: src/questions/sherlock.yaml (hunk 3 above)
Comment (mviswanathsai) [reply src/questions/sherlock.yaml]: Yes, I intended it to be like that, should I change it?

File: src/questions/sherlock.yaml (hunk 4 above)
Comment (mviswanathsai) [reply src/questions/sherlock.yaml]: Upto 50% for one single claim or multiple claims, but never more than 50%
I have changed the text for better clarity now

**Pull request merged**